src/main/java/az/edu/turing/happyfamilyv1/dao/impl/CollectionFamilyDao.java

```diff
@@ -76,7 +76,7 @@ public Family adoptChild(Family family, Human child) {
     public void deleteChildrenOlderThen(int age) {
         int currentYear = LocalDate.now().getYear();
         for (Family family : getAllFamilies()) {
-            family.getChildren().removeIf(child -> currentYear - child.getBirthYear()> age);
+            family.getChildren().removeIf(child -> currentYear - child.getYear()> age);
```

Comment:
this not works properly, current year as year format, but child.getYear() is millis. You are trying subtract millis from year